src/Illuminate/Encryption/Encrypter.php

```diff
@@ -33,16 +34,23 @@ class Encrypter implements EncrypterContract
      * Create a new encrypter instance.
      *
      * @param  string  $key
-     * @param  string|null  $cipher
+     * @param  string  $cipher
      * @return void
      */
-    public function __construct($key, $cipher = null)
+    public function __construct($key, $cipher = 'AES-128-CBC')
     {
-        $this->key = (string) $key;
+        $len = mb_strlen($key = (string) $key, '8bit');
```

Comment:
[You don't want mcrypt anyway](https://paragonie.com/blog/2015/05/if-you-re-typing-word-mcrypt-into-your-code-you-re-doing-it-wrong) ;)